opshin/tests/test_misc.py

```diff
@@ -939,3 +939,15 @@ def validator(xs: Dict[bytes, Dict[bytes, int]]) -> int:
             sum(v for pid, d in xs.items() for nam, v in d.items()),
             "for loop deconstruction did not behave as expected",
         )
+
+    def test_no_return_annotation_no_return(self):
+        source_code = """
+from opshin.prelude import *
+
+def validator(a):
+    pass
+"""
```

Comment:
Does this work with returning anything other than None?

Reply:
There are two lines here :)

> pass

This works with -> None or -> Anything (which is implicitly the case if you omit the return type)

> validator(a): 

This works with any return statement and no return statement (which implicitly returns None)